src/app/core/user/user.service.ts

```diff
@@ -19,53 +19,52 @@ export class UserService {
     return this._userDoc.asObservable();
   }
 
-  constructor(private authService: AuthService,
-              private userinfoService: UserinfoService) {
+  constructor(private authService: AuthService, private userinfoService: UserinfoService) {
     this.loadUser();
   }
 
   loadUser() {
     this.userinfoService.config(this.authHost);
-    this.authService.token.pipe(
-      first(),
-      concatMap(token => {
-        if (token) {
-          return combineLatest(of(token), this.userinfoService.getUserinfo());
-        } else {
-          return of([]);
-        }
-      }),
-      concatMap(([token, userinfo]) => {
-        if (token) {
-          this.loggedIn = true;
-          this.authorized = this.authService.parseToken(token);
-        } else {
-          this.loggedIn = false;
-        }
-        if (userinfo) {
-          this.userinfo = userinfo;
-          return this.userinfoService.getUserDoc(userinfo);
-        } else {
-          return of();
-        }
-      })
-    ).subscribe(doc => {
-      this._userDoc.next(doc);
-    });
+    this.authService.token
+      .pipe(
+        first(),
+        concatMap(token => {
+          if (token) {
+            return combineLatest([of(token), this.userinfoService.getUserinfo()]);
```

Comment:
a bunch of prettier whitespace changes in this file along with this update to combineLatest parameters being an array for newer RxJS v6.5